brainstormforce/header-footer-elementor
# Fix: "Can't read property querySelector of null" error due to conflict with Astra Header

### Description
Due to the same header ID selector, we are using as applied in Astra, the console error appears when the HFE header is applied on site. 

### Screenshots
https://share.getcloudapp.com/YEuXKbqj

### Types of changes

Bug fix (non-breaking change which fixes an issue)



### How has this been tested?
- Install and activate the Astra on your site
- Visit Dashboard -> Appearance -> Header Footer & Blocks
- Add a new template and assign a type "Header" Link
- Edit with Elementor
- Same error appears in the console

### Checklist:
